lib/coverage.js

```diff
@@ -437,10 +453,6 @@ internals.file = function (filename, data, options) {
             source: line
         };
 
-        if (options.sourcemaps) {
-            internals.addSourceMapsInformation(ret, +num);
-        }
```

Comment:
Move to the end of the loop, because we need to have the chunk before adding sourcemaps information